jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat

```diff
@@ -1,18 +1,20 @@
-@REM
-@REM Copyright © 2010-2014 Nokia
-@REM
-@REM Licensed under the Apache License, Version 2.0 (the "License");
-@REM you may not use this file except in compliance with the License.
-@REM You may obtain a copy of the License at
-@REM
-@REM      http://www.apache.org/licenses/LICENSE-2.0
-@REM
-@REM Unless required by applicable law or agreed to in writing, software
-@REM distributed under the License is distributed on an "AS IS" BASIS,
-@REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-@REM See the License for the specific language governing permissions and
-@REM limitations under the License.
-@REM
-
-@echo off
-java -jar "%~dp0/../lib/${project.build.finalName}.jar" %*
+@REM
+@REM Copyright © 2010-2014 Nokia
+@REM
+@REM Licensed under the Apache License, Version 2.0 (the "License");
+@REM you may not use this file except in compliance with the License.
+@REM You may obtain a copy of the License at
+@REM
+@REM      http://www.apache.org/licenses/LICENSE-2.0
+@REM
+@REM Unless required by applicable law or agreed to in writing, software
+@REM distributed under the License is distributed on an "AS IS" BASIS,
+@REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@REM See the License for the specific language governing permissions and
+@REM limitations under the License.
+@REM
+
+@echo off
+
+java -jar "%~dp0\${project.build.finalName}.jar" %*
+
```

Comment:
It looks like you've changed the CRLF line endings in this file to LF. Since this is a windows batch file, we need to keep the CRLF line endings. You could add this to you `.gitattributes` file, like:

```
*.bat text eol=crlf
```

Reply:
I believe I only did this for the .sh file and added a .gitattributes entry for it.

Reply:
You've actually swapped the CRLF line endings in this .bat file to LF as well (see the diff, all lines are changed).

Also, if you download a release from https://github.com/joelittlejohn/jsonschema2pojo/releases and look inside the archive, you'll see why this `../lib`` is required in these scripts. The scripts and the jars do not live alongside each other. We have brew integration that also depends on this layout.

Reply:
Thank you for directing me to the release download. I was fixing it for how the maven build would place it before you make your release but now it make much more sense.

It won't run as is in the build but that is not as important as the release.  I will revert these changes.